kymatio/scattering2d/tests/test_torch_scattering2d.py

```diff
@@ -0,0 +1,380 @@
+""" This script will test the submodules used by the scattering module"""
+
+import os
+import numpy as np
+import torch
+import pytest
+from kymatio.scattering2d import Scattering2D_torch as Scattering2D
+
+
+
+backends = []
+
+try:
+    if torch.cuda.is_available():
+        from skcuda import cublas
+        import cupy
+        from kymatio.scattering2d.backend import torch_skcuda_backend
+        backends.append(torch_skcuda_backend)
+except:
+    pass
+
+try:
+    from kymatio.scattering2d.backend import torch_backend
+    backends.append(torch_backend)
+except:
+    pass
+
+
+if torch.cuda.is_available():
+    devices = ['cuda', 'cpu']
+else:
+    devices = ['cpu']
+
+
+# Checked the modulus
+class TestModulus:
```

Comment:
are the classes necessary?

Reply:
cf above

Reply:
where? I think we managed to do very well without this overhead